src/runtime/providers/vercel.ts

```diff
@@ -1,15 +1,15 @@
-import { logDebug, getConnectionSpeed, send } from '../utils'
+import { RouteContext, WebVitalsMetric } from "../types"
+import { send } from "../utils"
+import { ModuleOptions } from '../../module'
```

Comment:
Please use `import type`:

```suggestion
import type { ModuleOptions } from '../../module'
```